source/settings/core.rst

```diff
@@ -2459,8 +2459,12 @@ See :ref:`settings` for list of all setting groups.
 
    A value of ``0`` means this scan never occurs.
 
+   .. versionchanged:: v2.3.21 In order to prevent load spikes the actual
+      value of the setting is spread increasing it by 0..30%, based on a hash
+      of the username.
+
    The scanning is done only for these mailbox formats:
-   
+
```

Comment:
extra whitespace.

Reply:
it was there previously and i removed it